### PR TITLE
Docu for /decodetrack & /decodetracks

### PR DESCRIPTION
**Pull request by topi314**

### CODE REVIEW

File: IMPLEMENTATION.md [hunk 1]
```diff
@@ -352,6 +352,66 @@ A severity level of `COMMON` indicates that the error is non-fatal and that the
 }
 ```
 
+### Decode Track API
```
Comment (freyacodes) [inline IMPLEMENTATION.md]:
```suggestion
### Track Decoding API
```

For consistency

File: IMPLEMENTATION.md [hunk 2]
```diff
@@ -352,6 +352,66 @@ A severity level of `COMMON` indicates that the error is non-fatal and that the
...
+
+Decode a single track into its info
+```
+GET /decodetrack?track=QAAAjQIAJVJpY2sgQXN0bGV5IC0gTmV2ZXIgR29ubmEgR2l2ZSBZb3UgVXAADlJpY2tBc3RsZXlWRVZPAAAAAAADPCAAC2RRdzR3OVdnWGNRAAEAK2h0dHBzOi8vd3d3LnlvdXR1YmUuY29tL3dhdGNoP3Y9ZFF3NHc5V2dYY1EAB3lvdXR1YmUAAAAAAAAAAA== HTTP/1.1
+Host: localhost:8080
```
Comment (freyacodes) [inline IMPLEMENTATION.md:360]: I think the documentation of the headers and HTTP version like this is pretty meaningless, but I'll accept it out of consistency with the existing documentation.

File: IMPLEMENTATION.md [hunk 3]
```diff
@@ -352,6 +352,66 @@ A severity level of `COMMON` indicates that the error is non-fatal and that the
...
+    "position": 0,
+    "title": "Rick Astley - Never Gonna Give You Up",
+    "uri": "https://www.youtube.com/watch?v=dQw4w9WgXcQ"
+  }
+}
+```
+
+Decode multiple tracks into info their info
+```
+GET /decodetracks HTTP/1.1
```
Comment (freyacodes) [inline IMPLEMENTATION.md]:
```suggestion
POST /decodetracks HTTP/1.1
```

**Review by freyacodes — CHANGES_REQUESTED** [review]

File: IMPLEMENTATION.md (hunk 2 above)
Comment (topi314) [reply IMPLEMENTATION.md:360]: thats the only reason I did it :) 👍🏻

**Review by freyacodes — APPROVED** [review]

**Pull request merged**